marconi-chain-index/src/Marconi/ChainIndex/Experimental/Indexers.hs

```diff
@@ -69,13 +71,13 @@ buildIndexers securityParam catchupConfig utxoConfig mintEventConfig epochStateC
     epochStateBuilder securityParam catchupConfig epochStateConfig mainLogger path
 
   StandardWorker utxoMVar utxoWorker <-
-    utxoBuilder securityParam catchupConfig utxoConfig txBodyCoordinatorLogger path
+    utxoBuilder securityParam catchupConfig utxoConfig txBodyCoordinatorLogger textLogger path
```

Comment:
Don't like that part of passing two different loggers here. Suggestions accepted.